pom.xml

```diff
@@ -38,53 +37,38 @@
         <connection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-jcr-registration.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-jcr-registration.git</developerConnection>
         <url>https://gitbox.apache.org/repos/asf?p=sling-org-apache-sling-jcr-registration.git</url>
-      <tag>HEAD</tag>
-  </scm>
+    </scm>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <extensions>true</extensions>
-                <configuration>
-                    <instructions>
-                        <Bundle-Category>
-                            sling,jcr
-                        </Bundle-Category>
-                        <Import-Package>
-                            javax.transaction.xa;resolution:=optional,
-                            *
-                        </Import-Package>
-                        <DynamicImport-Package>
-                            *
-                        </DynamicImport-Package>
-                        <Embed-Dependency>
-                            jackrabbit-core;inline=org/apache/jackrabbit/core/jndi/provider/*.class
-                        </Embed-Dependency>
-                    </instructions>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
+    <properties>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
+    </properties>
 
     <dependencies>
```

Comment:
please repeat the scope from depMgmt, to make it clear that all of them are `provided`

Reply:
Fixed